sda/host.go

```diff
@@ -612,3 +617,63 @@ func (h *Host) Rx() uint64 {
 func (h *Host) Address() string {
 	return h.workingAddress
 }
+
+//Received gives packets received
+func Received(n map[network.ServerIdentityID]network.SecureConn) uint64 {
+	var a uint64
+	for _, value := range n {
+		a = value.Rx()
+	}
+	return a
```

Comment:
This doesn't really look correct. Isn't this supposed to add/aggreagte the packages received? Here it just returns the last measurement found in the map.